FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/Demos.kt

```diff
@@ -167,20 +167,20 @@ val V2DEMO = arrayListOf(
     Demo(V2BASIC_CHIP, V2BasicChipActivity::class),
     Demo(V2BASIC_CONTROLS, V2BasicControlsActivity::class),
     Demo(V2BOTTOM_DRAWER, V2BottomDrawerActivity::class),
-    Demo(V2BOTTOM_SHEET, V2BottomSheetActivity::class, Badge.APIBreak),
+    Demo(V2BOTTOM_SHEET, V2BottomSheetActivity::class),
     Demo(V2BUTTON, V2ButtonsActivity::class),
     Demo(V2CARD, V2CardActivity::class),
-    Demo(V2CARD_NUDGE, V2CardNudgeActivity::class, Badge.Modified),
+    Demo(V2CARD_NUDGE, V2CardNudgeActivity::class),
     Demo(V2CITATION, V2CitationActivity::class),
     Demo(V2CONTEXTUAL_COMMAND_BAR, V2ContextualCommandBarActivity::class),
     Demo(V2DIALOG, V2DialogActivity::class),
-    Demo(V2DRAWER, V2DrawerActivity::class),
+    Demo(V2DRAWER, V2DrawerActivity::class, Badge.Modified),
     Demo(V2LABEL, V2LabelActivity::class),
-    Demo(V2LIST_ITEM, V2ListItemActivity::class),
+    Demo(V2LIST_ITEM, V2ListItemActivity::class, Badge.APIBreak),
```

Comment:
shouldnt it be modified

Reply:
API has changed, before this there was no parameter for onClick, now it has and also default behavior has changed